pkgs/development/python-modules/pysensors/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, lm_sensors }:
+buildPythonPackage rec {
+  version = "2018-04-15";
+  pname = "pysensors";
+
+  src = fetchFromGitHub {
```

Comment:
Please add a comment that https://pypi.org/project/PySensors/ is a different package.

Reply:
Above `src`?

Reply:
Yes